tox.ini

```diff
@@ -1,6 +1,8 @@
+# Operator charm (with zaza): tox.ini
```

Comment:
This should probably be source primarily from [the globally shared tox.ini](https://github.com/openstack-charmers/release-tools/blob/master/global/source-zaza/tox.ini). Whic hit possibly is, but if so - it needs an update.